solidity/test/BondedECDSAKeepFactoryTest.js

```diff
@@ -1,32 +1,35 @@
-import {createSnapshot, restoreSnapshot} from "./helpers/snapshot"
+const {accounts, contract, web3} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
 
 const {expectRevert, time} = require("@openzeppelin/test-helpers")
 
-import {mineBlocks} from "./helpers/mineBlocks"
+const {mineBlocks} = require("./helpers/mineBlocks")
 
 const truffleAssert = require("truffle-assertions")
 
-const KeepRegistry = artifacts.require("KeepRegistry")
-const BondedECDSAKeepFactoryStub = artifacts.require(
+const StackLib = contract.fromArtifact("StackLib")
```

Comment:
What is `StackLib`? I cannot find it anywhere.

Reply:
[StackLib](https://github.com/keep-network/sortition-pools/blob/master/contracts/StackLib.sol)

You'd get this if we won't link it:
```
Contract contains unresolved libraries. You must deploy and link the following libraries before you can deploy a new version of Contract: StackLib
```

Reply:
Right, it's from sortition pools. I have no better idea of how to solve it, let's leave it as-is.

Reply:
In `2_deploy_contract.js` we solve all the linkage problems with
```
const {
  deployBondedSortitionPoolFactory,
} = require("@keep-network/sortition-pools/migrations/scripts/deployContracts")
const BondedSortitionPoolFactory = artifacts.require(
  "BondedSortitionPoolFactory"
)
```

but it's not that easy here. That's fine, let's leave it as-is and approach it again in future in case we'll have any better idea.